src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs

```diff
@@ -80,8 +81,12 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                                 throw new Exception(result.Details);
                             }
 
+                            List<AIReviewComment> validComments = result.Comments?
+                                .Where(comment => jobInfo.CodeLines[comment.LineNo - 1].lineId != null)
+                                .ToList() ?? new List<AIReviewComment>();
```

Comment:
We need to make sure that this change does not result in the summary comment not being displayed.

Reply:
We are adding this on the fly though https://github.com/Azure/azure-sdk-tools/blob/bc139adb151ac140bf26907ece605ce864544517/src/dotnet/APIView/APIViewWeb/Helpers/CodeFileHelpers.cs#L347 on APIView side rather than the parser. So you should definitely add the check for summary.